controllers/team.controller.js

```diff
@@ -7,6 +7,30 @@ const Services = {
 };
 const Util = require("../middlewares/util.middleware");
 
+/**
+ * @async
+ * @function findById
+ * @param {*} req 
```

Comment:
what are you expecting in this req object

Reply:
Deferred